database/cassandra/index.js

```diff
@@ -0,0 +1,52 @@
+const cassandra = require('cassandra-driver');
+
+// https://docs.datastax.com/en/developer/nodejs-driver/4.4/features/connection-pooling/
+
+const client = new cassandra.Client({
+  contactPoints: ['127.0.0.1', '127.0.0.1'],
+  localDataCenter: 'datacenter1',
+  keyspace: 'etsy_v02',
+  // pooling: {
+  //   maxRequestsPerConnection: 20000,
+  // },
+});
+
+const t0 = new Date();
+
+
+client.connect()
+  .then(() => {
```

Comment:
every time you connect, it's going to run a query?

Reply:
Yes, this script was for testing purposes.